tephra-core/src/main/java/org/apache/tephra/runtime/TransactionDistributedModule.java

```diff
@@ -35,13 +34,24 @@
 import org.apache.tephra.persist.TransactionStateStorage;
 import org.apache.tephra.snapshot.SnapshotCodecProvider;
 
+import java.lang.management.ManagementFactory;
+
 /**
  * Guice bindings for running in distributed mode on a cluster.
  */
-final class TransactionDistributedModule extends AbstractModule {
+final class TransactionDistributedModule extends ClientIdAwareTransactionModule {
+
+  public TransactionDistributedModule() {
+    super(ManagementFactory.getRuntimeMXBean().getName());
```

Comment:
In that case they can provide the client id since the default constructor won't be present, right?